administrator/language/en-GB/joomla.ini

```
@@ -241,6 +241,9 @@ JFIELD_LOGOUT_REDIRECT_PAGE_DESC="Select or create the page the user will be red
 JFIELD_LOGOUT_REDIRECT_PAGE_LABEL="Logout Redirection Page"
 JFIELD_LOGOUT_REDIRECT_URL_DESC="If a URL is entered here, users will be redirected to it after logout.<br>The URL must be internal (eg: index.php?Itemid=999)."
 JFIELD_LOGOUT_REDIRECT_URL_LABEL="Logout Redirect"
+JFIELD_MEDIA_ALT_LABEL="Alternative text"
```

```suggestion
JFIELD_MEDIA_ALT_LABEL="Alt Text"
```

To be consistent with COM_CONTENT_FIELD_IMAGE_ALT_LABEL